INSTALL.md

```diff
@@ -54,49 +54,32 @@ to stop the application, run:
 make dev-stop
 ```
 
-## Manual setup on Mac OS X (10.13 - 12.3.1)
+## Manual setup on Mac OS X (10.13 - 12.5.1)
 
 ### Prerequisites
 
-#### Ruby version
+#### Ruby and Node.js
 
-The project supports **2.6.x**.
-
-Verify you have a proper version by running on your terminal:
+The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 12](https://nodejs.org/en/download/)**. Verify you have a proper version by running on your terminal:
 
 ```bash
-ruby -v
+ruby -v && node -v
 ```
 
-> We recommend using [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install any ruby version.
+[asdf](https://asdf-vm.com/guide/getting-started.html#global) is a convenient tool version manager that reads
+from the `.tool-versions` file included in this project. This way you won't have to worry about the versioning
+of these packages and it will override the system's ruby version. After installing it using Homebrew you need
+to install a plugin for each individual package:
 
-Example with RVM:
-```bash
-gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-\curl -sSL https://get.rvm.io | bash -s stable
-rvm install 2.6
+```sh
+brew install asdf
+asdf plugin add ruby https://github.com/asdf-vm/asdf-ruby.git
```

Comment:
The URL is optional, so can be just `asdf plugin add ruby`, and the same for nodejs